src/domains/services/validation.service.ts

```diff
@@ -59,16 +62,76 @@ const validFormats = [
   'pretty',
 ];
 
+type AuthEntry = {
+  pattern: string;
+  token: string;
+};
+
+const CONFIG_FILE = path.join(os.homedir(), '.asyncapi', 'config.json');
```

Comment:
Can't we import this from somewhere else? I don't think it's the responsibility of the ValidationService to know how to construct the config directory.

Reply:
Make sense i will do : )

Reply:
Created a new service thanks : )